index.js

```diff
@@ -152,7 +153,7 @@ function createSeries (opts, archs, platforms) {
                 })
               } else {
                 console.error('Skipping ' + platform + ' ' + arch +
-                  ' (output dir already exists, use --overwrite to force)')
+                    ' (output dir already exists, use --overwrite to force)')
```

Comment:
Oops sorry about that, it must have been something with my IDE that I overlooked